cmd/getConfig_test.go

```diff
@@ -54,8 +54,7 @@ func TestConfigCommand(t *testing.T) {
 	t.Run("Run", func(t *testing.T) {
 		t.Run("Success case", func(t *testing.T) {
 			configOptions.openFile = configOpenFileMock
-			err := cmd.RunE(cmd, []string{})
-			assert.NoError(t, err, "error occured but none expected")
+			cmd.Run(cmd, []string{})
```

Comment:
I don't see how the test still performs the intended check. Can you clarify?

Reply:
The program will exit with an error in case of a failure, thus the test should still be valid.